airflow/contrib/example_dags/example_gcp_bigtable_operators.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# 'License'); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that creates and performs following operations on Cloud BigTable:
+- creates an Instance
+- creates a Table
+- updates Cluster
+- waits for Table replication completeness
+- deletes the Table
+- deletes the Instance
+
+This DAG relies on the following environment varialbles
+* PROJECT_ID - Google Cloud Platform project.
+* INSTANCE_ID - desired ID of an BigTable instance
+* INSTANCE_DISPLAY_NAME - desired human-redable display name of the Instance
+* INSTANCE_TYPE - type of the Instance, e.g. 1 for DEVELOPMENT
+    SeeSee https://googleapis.github.io/google-cloud-python/latest/bigtable/instance.html#google.cloud.bigtable.instance.Instance
+* INSTANCE_LABELS - labels to add for the Instance
+* CLUSTER_ID - desired ID of the main Cluster created for the Instance
+* CLUSTER_ZONE - zone in which main Cluster will be created. e.g. europe-west1-b
+    See available zones: https://cloud.google.com/bigtable/docs/locations
+* CLUSTER_NODES - initial amount of nodes of the Cluster
+* CLUSTER_NODES_UPDATED - amount of nodes for BigTableClusterUpdateOperator
+* CLUSTER_STORAGE_TYPE - storage for the Cluster, e.g. 1 for SSD
+    See
+* REPLICA_CLUSTER_ID - desired ID of the replica Cluster created for the Instance. Applicable only for PRODUCTION
+    instances
+* REPLICA_CLUSTER_ZONE - zone in which replica Cluster will be created. Should be different than CLUSTER_ZONE,
+    yet in the same region - e.g. europe-west1-d
+* TABLE_ID - desired ID of the Table
+"""
+
+import datetime
+import json
+
+from os import getenv
+
+import airflow
+from airflow import models
+from airflow.contrib.operators.gcp_bigtable_operator import BigTableInstanceCreateOperator, \
+    BigTableInstanceDeleteOperator, BigTableClusterUpdateOperator, BigTableTableCreateOperator, \
+    BigTableTableWaitForReplicationSensor, BigTableTableDeleteOperator
+
+# [START howto_operator_gcp_bigtable_args]
+PROJECT_ID = getenv('CBT_PROJECT_ID', 'example-project')
```

Comment:
In other places I renamed also the variable names. It might be good to keep it consistent.